src/layouts/StarWarsIntro/StarWarsIntro.js

```diff
@@ -0,0 +1,70 @@
+import Header from '../../components/header/Header';
+import Container from '../../components/container/Container';
+import Description from '../../components/description/Description';
+import './styles.css';
+
+class StarWarsIntro {
+  render() {
+    const container = document.createElement('section');
+
+    container.innerHTML = '';
+    container.classList.add('intro__container');
+
+    const containerElement1 = new Container(['intro'], 'div');
```

Comment:
As there is no `containerElement2`, maybe it could be renamed descriptively without using a number in the name?